playbooks/cloud-provider-openstack-acceptance-test-csi-manila/run.yaml

```diff
@@ -0,0 +1,201 @@
+- hosts: all
+  roles:
+    - config-golang
+    - export-cloud-openrc
+    - install-k8s
+  become: yes
```

Comment:
Any reason we want to test this with a real cloud provider rather than with DevStack? Setup/run time?

The pre-setup you added to the current manila-provisioner acceptance test job should work here in case we could use DevStack:

https://github.com/theopenlab/openlab-zuul-jobs/blob/f12baea228cdbcc4186354d0e3e2f81b06c778bd/playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml#L6-L14

Reply:
Just quicker test runs

@gouthampacha and I just discussed this offline. The Citynetwork cloud doesn't seem to offer Manila anyway so we're going for a DevStack deployment